sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/SyncAccountRepository.kt

```diff
@@ -882,11 +901,11 @@ enum class AccountErrorCodes(val code: Int) {
     EXCHANGE_FAILED(56),
 }
 
-enum class CodeType {
-    RECOVERY,
-    CONNECT,
-    EXCHANGE,
-    UNKNOWN,
+sealed interface CodeType {
```

Comment:
when reading this in another class `syncAccountRepository.processCode(codeType)` my first thought was "you can pass the type but where's the code?" 

So I maybe codeType should be something else, trying to think... probably `SyncAuthCode`?

Reply:
`SyncAuthCode` works for me